src/main/java/uk/gov/hmcts/reform/jps/model/out/SittingRecord.java

```diff
@@ -46,6 +57,23 @@ public class SittingRecord {
     @ToString.Exclude
     private List<StatusHistory> statusHistories;
 
+    public String getCreatedByUserId() {
+        StatusHistory statusHistory = getFirstStatusHistory();
+        return null != statusHistory ? statusHistory.getChangeByUserId() : null;
+    }
+
+    @JsonIgnore
+    public StatusHistory getFirstStatusHistory() {
+        if (null == statusHistories) {
```

Comment:
Could replace with - statusHistories.stream().min(Comparator.comparingLong(StatusHistory::getId)).orElse(null);

Reply:
Cool

Reply:
This is still pending

Reply:
Done